.travis.yml

```diff
@@ -22,9 +22,11 @@ addons:
     postgresql: '9.5'
```

Comment:
```suggestion
    postgresql: '12'
```

Might as well

Reply:
Well ideally you should be testing with the oldest version supported. Is there any particular need to bump it further?

Reply:
I'm not too fussed, if we keep it at the lowest, it'll be obvious when new features aren't compatible with the lowest version. Seems sensible.

Reply:
I did run into some incompatibilities when doing the 10->12 upgrade wrt. `WITH OIDS`, but not sure if that was a quirk of `pg_dump`'s output (read: I forgot some important flag somewhere), or actual breaking changes that'd bork `bsae.sql`.

Reply:
Ah yes, `base.sql` uses `default_with_oids`, which is deprecated since 11, and explicit `WITH OIDS` is gone in 12. I don't think we use them anywhere though.